### ISSUE TITLE
Old version on RubyGems

### ISSUE THREAD

**Issue by rod-murphy** [phstc/param_store#4]
I'd really like to try out this gem for a project, however, the code in the `master` branch isn't yet live on [rubygems](https://rubygems.org/gems/param_store) have you any plans to release it?

**Comment by phstc** [comment]
Hi @rod-murphy 

Thanks for checking this. I've been using the master version for a while 

```ruby
gem 'param_store', git: 'https://github.com/phstc/param_store'
```

it is stable 🙏 

I forgot to bump it on RubyGems. [I've just released 1.0.0](https://rubygems.org/gems/param_store). Please let me know if that works for you.

**Comment by rod-murphy** [comment]
Brilliant. Thanks. I'll take it for a spin and report back 👍
